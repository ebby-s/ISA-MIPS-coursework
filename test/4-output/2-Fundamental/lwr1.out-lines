TB : INFO : register_v0=07040201
TB : finished; active=0
