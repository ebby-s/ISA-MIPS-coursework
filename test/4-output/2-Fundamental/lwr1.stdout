Loading test/2-binary/2-Fundamental/lwr1.hex.txt into RAM
VCD info: dumpfile mips_cpu_bus_tb.vcd opened for output.
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x3c100704
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x26100201
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0xac10000c
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x9802000c
TB : INFO : RAM_ACCESS: Write to 0x0000000c, data: 0x07040201
TB : INFO : RAM_ACCESS: Read from 0xbfc00010, data: 0x03e0f808
TB : INFO : RAM_ACCESS: Read from 0x0000000c, data: 0x07040201
TB : INFO : RAM_ACCESS: Read from 0xbfc00014, data: 0x00000000
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x00000000
TB : INFO : register_v0=07040201
TB : finished; active=0
